runtime/shibuya/src/xcm_config.rs

```diff
@@ -260,3 +270,53 @@ impl cumulus_pallet_dmp_queue::Config for Runtime {
     type XcmExecutor = XcmExecutor<XcmConfig>;
     type ExecuteOverweightOrigin = EnsureRoot<AccountId>;
 }
+
+/// Convert `AccountId` to `MultiLocation`.
+pub struct AccountIdToMultiLocation;
+impl Convert<AccountId, MultiLocation> for AccountIdToMultiLocation {
+    fn convert(account: AccountId) -> MultiLocation {
+        X1(AccountId32 {
+            network: None,
+            id: account.into(),
+        })
+        .into()
+    }
+}
+
+parameter_types! {
+    /// The absolute location in perspective of the whole network.
+    pub ShibuyaLocationAbsolute: MultiLocation = MultiLocation {
+        parents: 1,
+        interior: X1(
+            Parachain(ParachainInfo::parachain_id().into())
+        )
+    };
+    pub const MaxAssetsForTransfer: usize = 2;
```

Comment:
Is there some reasoning behind choosing the `MaxAssetsForTransfer` value, or we are just trying to be cautious? Code comment explaining that would help if there is some reasoning behind this, otherwise please ignore this?

Reply:
It's just to be cautious. For one thing, we don't have use cases to transfer more than 2 assets. For another, the test coverage in XCM executor and `orml-xtokens` are mainly transferring one or two assets. I'll add a comment docstring.